File: CS50/Week-6/mario.py
# TODO

def main():
    height = 0
    while height < 1 or height > 8:
        height = int(input("Enter height: "))

    for rows in range(height):
        # Print spaces before the hashes to make a right-aligned pyramid
        for spaces in range(height - rows - 1):
            print(" ", end="")

        # Print hashes for each column
        for columns in range(rows + 1):
            print("#", end="")

        # Print the gap between the two sides of the pyramid
        print("  ", end="")

        # Print the hashes for the second side of the pyramid
        for columns in range(rows + 1):
            print("#", end="")

        print()  # Move to the next line

if __name__ == "__main__":
    main()

File: CS50/Week-1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height, rows, columns, spaces;
    do
    {
        printf("enter height: ");
        scanf("%i", &height);
    }
    while(height >8 || height < 1);
    //for each row//

    for (rows=0; rows <  height ; rows++)
    {
        // print spaces before the hashes to make a left-aligned pyramid//
        for(spaces=0; spaces< height-rows-1; spaces++)
        {
            printf(" ");
        }
        //for each column//
        for(columns=0; columns<= rows; columns++)
            {
                 printf("#");
            }
            printf("\n");

    }

}